### DISCUSSION TITLE
how to know if the alert banner

### DISCUSSION THREAD

**Discussion by ofirgo1** [KevinVandy/mantine-react-table#369] [Q&A]
### mantine-react-table version

1.3.4

### react & react-dom versions

18.2.0

### Describe the bug and the steps to reproduce it

i added to my table the option to group by column.
when use it, the table going out from my screen because the alert banner pushing it down.
i want to change the height of my table when the alert banner open, but i cant find any solution for my issue.
I would appreciate it if someone could explain to me if there is a way to solve this problem.
(I could not use the showAlertBanner state, if it is possible to use it then I would appreciate a good explanation) 

### Minimal, Reproducible Example - (Optional, but Recommended)

.

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment]
It's stored in the table state, so you can read or set that state just like any other state in the table.
